app/assets/stylesheets/osem.css.scss

```diff
@@ -71,4 +71,14 @@ p.comment-body {
 
 #account-already {
   font-size: 0.6em;
+}
+
+/* comments views pane: use padding-bottom before next comment box */
+.panel.panel-default .panel-body .box {
+  padding-bottom: 20px;
+}
+
+/* comments views pane: use padding-bottom after each comment */
+.panel.panel-default .panel-body .box p {
```

Comment:
Merge rule `.panel.panel-default .panel-body .box p` with rule on line 77<br>Selector should have depth of applicability no greater than 2, but was 4